python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
+    num = 0
     for letter in text:
         if letter not in vowels:
-            print("consonant", letter)
+           num += text.lower().count(letter)
```

Comment:
Because the for loop is iterating through each letter in the input sentence (_text_) and checking whether it is not a vowel, the num variable should increase by 1.
```suggestion
           num += 1
```